properties/P000024.md

```diff
@@ -10,7 +10,7 @@ refs:
     name: Locally compact space on Wikipedia
 ---
 
-Each point has a basis of neighborhoods relatively compact to the space.
+Each point has a local base of neighborhoods relatively compact to the space (that is, neighborhoods with compact closure).
```

Comment:
I generally call such subsets "precompact" and reserve "relatively compact" for "every open cover of the *space* has a finite subcover for the *subset*". These are equivalent assuming regularity, but not in general.

So if the intent here is actually "a local base of neighborhoods with compact closures", then I think we should just say that and avoid the ambiguity.

Reply:
I made this change and approved, so go ahead and merge if you're happy with the tweak.

Reply:
Hmm I guess the name "locally relatively compact" is unclear as well, but maybe that can be handled separate from the PR.